lib/utils/cache.js

```diff
@@ -79,8 +90,39 @@ module.exports = class Cache {
         key,
         value: SafeJSON.stringify(value)
       });
-    } catch (err) {
-      return;
-    }
+    } catch (err) { /* passthrough */ }
   }
-};
+
+  /**
```

Comment:
@tptee -- I'm not 100% satisfied with this approach, but what I'm trying to do is:

1. Give us a simple wrapper for our always use case of "check cache, do action, set cache"
2. Not call the potentially expensive `hash()` function unless we're actually doing cache get/set. (That's why `NoopCache.wrapAction` notably _doesn't_ call any hash function).

Thoughts?